### ISSUE TITLE
Mongoose: Failed to load c++ `bson` extension...

### ISSUE THREAD

**Issue by ClaytonSmith** [ClaytonSmith/Pcaso#7]
This is an internal issue with Mongoose. It does not appear to affect this project but a warning message is displayed when a mongoose instance is created. 

Error: 
```
{ [Error: Cannot find module '../build/Release/bson'] code: 'MODULE_NOT_FOUND' }
js-bson: Failed to load c++ bson extension, using pure JS version
```

**Comment by ClaytonSmith** [comment]
It fixed itself somehow. That's cool :+1:

**Comment by ClaytonSmith** [comment]
Issue still exists. NVM reset istelf, setting the default Node version to an older version. This issue does not exist in older versions on Node. Node v5 and up is needed for this project.

**Comment by ClaytonSmith** [comment]
Fixed by upgrading to v5.4.1.
